tests/test_RBAC.py

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+
+import pytest
+from ocean_lib.common.agreements.service_types import ServiceTypes
+from ocean_lib.models.data_token import DataToken
+
+from ocean_provider.constants import BaseURLs
+from ocean_provider.exceptions import RequestNotFound
+from ocean_provider.utils.accounts import generate_auth_token
+from ocean_provider.validation.algo import build_stage_output_dict
+from ocean_provider.validation.requests import RBACValidator
+from tests.helpers.compute_helpers import build_and_send_ddo_with_compute_service
+from tests.test_helpers import (
+    get_dataset_ddo_with_access_service,
+    mint_tokens_and_wait,
+    send_order,
+)
+
+
+def test_null_validator():
+    with pytest.raises(RequestNotFound):
```

Comment:
why does this still raise "RequestNotFound"? RBAC Validator is an abc now, right? It shouldn't be instantiated

Reply:
please transform this into the proper RequestNotFound test (missing action/request_name from action mapping)